docs/requirements.txt

```diff
@@ -1,3 +1,3 @@
-sphinx
+sphinx==5.3.0
```

Comment:
#1239 was created to update this to 6.2.1

Reply:
true I mixed up the conflicts. thanks